vizro-core/tests/unit/vizro/models/_controls/test_filter.py

```diff
@@ -132,6 +255,13 @@ def test_set_slider_values_defaults_min_max_none(self, test_input, gapminder, ma
         assert filter.selector.min == gapminder.lifeExp.min()
         assert filter.selector.max == gapminder.lifeExp.max()
 
+    def test_set_datepicker_values_defaults_min_max_none(self, gapminder_with_datetime, managers_one_page_two_graphs):
+        filter = vm.Filter(column="year", selector=vm.DatePicker())
+        model_manager["test_page"].controls = [filter]
+        filter.pre_build()
+        assert filter.selector.min == gapminder_with_datetime.year.min().to_pydatetime().date()
```

Comment:
Silly question, but where does it get it's min and max values. I could only find it for the slider....

Reply:
Inside the `Filter._set_date_picker_values` method, that is called inside the `Filter._pre_build` phase.